gc/gc-base/src/main/java/org/projectnessie/gc/base/GCParams.java

```diff
@@ -70,6 +70,20 @@ default double getBloomFilterFpp() {
     return 0.03d;
   }
 
+  /** Nessie catalog name to be used with spark to create the output results table. */
+  String getNessieCatalogName();
+
+  /**
+   * Reference name to be used for creating the output table.
+   *
+   * <p>If the reference doesn't exist for this reference name, reference with this name pointing to
+   * beginning of time (aka NO_ANCESTOR hash) will be created.
+   */
+  String getOutputTableRefName();
```

Comment:
Not strictly related to this PR, but how do we handle weird table names and escaping stuff? Shouldn't we have a dedicated `TableRefName` type instead of raw `String`?

Reply:
Here, raw string is fine as it is just the reference name.

But internally for DDL and DML, we need to append  `@ref` and escape characters to the table name to tell SQL parser that it is associated with some reference. 
I just discovered that we have `TableReference` class, I will try to use it internally for DDL and DML